Extensions/TweenBehavior/JsExtension.js

```diff
@@ -179,7 +207,7 @@ module.exports = {
         _('Tween the camera rotation'),
         _('Tweens the camera rotation from the current angle to a new one.'),
         _(
-          'Tween the rotation of camera on layer _PARAM3_ to _PARAM2_ for _PARAM4_ms with easing _PARAM5_ as _PARAM1_'
+          'Tween the rotation of camera on layer _PARAM3_ to _PARAM2_ over _PARAM4_ms with easing _PARAM5_ as _PARAM1_'
```

Comment:
Just to confirm, does using `_PARAM4_ms` display "ms" in the editor?

Reply:
Yes!